### PR TITLE
Feature/player vars

### PR DESCRIPTION
**Pull request by clementoriol**
https://github.com/ginpei/html5-youtube.js/issues/5

@ginpei You must have misread my post, I started working on it yesterday :)
Still a WIP, I'd like to implement other settings as well

### CODE REVIEW

**Comment by clementoriol** [comment]
Should be mergeable.
Would love to have your opinion on it. :)

**Here's a quick review of the changes:**
- Allows extra player parameters when initializing the player (https://developers.google.com/youtube/player_parameters#Parameters)
- Initialize the YT player with the correct values (expects mostly numbers, not booleans)
- Update the tests
- Update the Readme

File: examples/playervars.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<title>youtube.js</title>
+	</head>
+	<body>
+		<h1>youtube.js playerVars settings</h1>
+		<p><a href="https://developers.google.com/youtube/player_parameters">https://developers.google.com/youtube/player_parameters</a></p>
```
Comment (ginpei) [inline examples/playervars.html:9]: I like kind things like this link here. :+1:

**Comment by ginpei** [comment]
I'm sorry for misunderstanding and thank you for your PERFECT work!

**Comment by clementoriol** [comment]
No worries, glad you liked it :+1: !

**Pull request merged**